src/components/modal/modal.scss

```diff
@@ -184,13 +189,44 @@
  */
 .content {
   @apply relative box-border block h-full overflow-auto p-0;
-  max-block-size: 100%;
   background-color: var(--calcite-modal-content-background, theme("colors.background.foreground.1"));
-  padding-block: var(--calcite-modal-content-padding, var(--calcite-modal-padding-internal));
-  padding-inline: var(--calcite-modal-content-padding, var(--calcite-modal-padding-internal));
+  max-block-size: 100%;
+  padding: var(--calcite-modal-content-padding, var(--calcite-modal-padding-internal));
+}
+
+.content-top {
+  @apply bg-foreground-1
+  border-color-3
+  flex
+  min-w-0
+  max-w-full
+  rounded-t
+  border-0
+  border-b
+  border-solid
+  z-header;
```

Comment:
Is `z-header` needed for the top and bottom areas?

Reply:
It's on the same level as the header, so I kept it.